### PR TITLE
MADE IN IRAN: Adding 2 new repositories

### PR DESCRIPTION
**Pull request by BaseMax**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [x] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/made-in-iran/index.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+ - KambizAsadzadeh/Project-Template
  - rastikerdar/vazirmatn
  - mohebifar/made-in-iran
  - Hameds/APIs-made-in-Iran
+ - sobhe/moratab
+ - jamedadi/yummy
+ - sobhe/hazm
+ - mr-hashemi/mr-hashemi
+ - PyFarsi/pyabr
+ - pashmaklang/pashmak
```
Comment (kenyonj) [inline collections/made-in-iran/index.md:15]: We aren't accepting re-order PRs at this time.

Comment (BaseMax) [reply collections/made-in-iran/index.md:15]: `KambizAsadzadeh/Project-Template` is a new one.

File: collections/made-in-iran/index.md [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
  - Kiarash-Z/react-modern-calendar-datepicker
  - MahdiMajidzadeh/bootstrap-v4-rtl
  - imaNNeoFighT/fl_chart
  - persian-tools/persian-tools
  - usablica/intro.js
 display_name: Made in Iran
-created_by: Javad
```
[the remark this answers is not in the document]
Comment (kenyonj) [reply collections/made-in-iran/index.md:20]: I believe it's the name of the user that created this collection originally
re: https://github.com/github/explore/pull/2535

File: collections/made-in-iran/index.md (hunk 1 above)
Comment (BaseMax) [reply collections/made-in-iran/index.md:15]: `jadijadi/linuxandlife` is a new one too.

Comment (kenyonj) [reply collections/made-in-iran/index.md:15]: sounds good, just append them to the end and we can review

Comment (BaseMax) [reply collections/made-in-iran/index.md:13]: There are the same authors, It's why I posted that at the top. By the way, I did what you asked.

**Pull request merged**